lookout/style/format/features.py

```diff
@@ -225,17 +247,95 @@ def _classify_vnodes(self, nodes: Iterable[VirtualNode]) -> List[VirtualNode]:
                             y=cls))
         return result
 
+    def _inplace_write_vnode_features(self, features: numpy.ndarray, vnode: VirtualNode, row: int,
+                                      vnode_index: int) -> None:
+        """
+        Given a feature `numpy.ndarray`, a `VirtualNode`, a row and a feature index, write the
+        features of the vnode at the correct location in the feature matrix.
+
+        :param features: `numpy.ndarray` the features matrix.
+        :param vnode: `VirtualNode` we want to write the features of.
+        :param row: on which row to write the features.
+        :param vnode_index: the column at which we'll start writing the features is
```

Comment:
Sphinx requires `\` when breaking the line for `:param` (if I remember correctly)